docs/mediawiki-loading.md

```diff
@@ -0,0 +1,22 @@
+## MediaWiki Loading
```

Comment:
It could make sense to call this something like `How WBStack custom code is loaded by Mediawiki`